src/drivers/mpu6000/mpu6000.h

```diff
@@ -204,12 +204,12 @@
 #define MPU6000_ACCEL_DEFAULT_DRIVER_FILTER_FREQ	30
 
 #define MPU6000_GYRO_DEFAULT_RANGE_G				8
-#define MPU6000_GYRO_DEFAULT_RATE					1000
+#define MPU6000_GYRO_DEFAULT_RATE					8000
```

Comment:
@jgoppert -  I think we would need to instrument the 8K rate, My recollection from work I did on the icm20608 @ 4k - #3360 was that the device Access/Processing with interrupts off was 600+ uS out of 1000 uS. I did not decompose that into access and processing but I suspect that DMA is needed and the driver would need to be reworked to have a short (DMA)/long service (HP work processing)